src/Components/NewNote/NewNote.jsx

```diff
@@ -125,7 +136,7 @@ const NewNote = () => {
       alertDispatch({ type: "hideInputWithData" });
     }
     setShowColor(false);
-    setNoteColor("#f0fbff");
+    setNoteColor("#ffffff");
```

Comment:
Suggestion: You could create a data file for variable names and use that instead of values directly here. That will help in changing the colors at a single place and more readable